### PR TITLE
issue/606/Fix displaying errors and warnings while making build

### LINKED ISSUE #606: Fix displaying errors and warnings while making build

### PR DESCRIPTION
**Pull request by dbetka**
close #606 

**Done:**
- Add validation for AppConsoleFramePlugin params
- Stop displaying progress bar while errors or warnings come
- Copy and improve progress-bar-webpack-plugin
- Restored writeAssetsSizes method
- Restored build time diplay
- Improve clear console method
- Merge ProgressBarConfig into AppConsoleFramePlugin
- **Move webpack plugins to wdk**

Documentation for webpack plugins:
https://harcmap.atlassian.net/wiki/spaces/HARCMAP/pages/100499525/App+Console+Frame

### CODE REVIEW

File: client/webpack/plugins/app-console-frame.js [hunk 1]
```diff
@@ -1,152 +0,0 @@
-const chalk = require('chalk');
```
Comment (Bedpaw) [inline client/webpack/plugins/app-console-frame.js:1]: Should we remove   "chalk": "^4.1.2"  from package.json?

File: client/webpack/plugins/app-console-frame.js [hunk 2]
```diff
@@ -1,152 +0,0 @@
...
-const dayjs = require('dayjs');
-dayjs.extend(require('dayjs/plugin/duration'));
-const { Table } = require('console-table-printer');
```
Comment (Bedpaw) [inline client/webpack/plugins/app-console-frame.js:4]: Should we remove  "console-table-printer": "^2.10.0", from package.json?

File: client/webpack/plugins/app-console-frame.js (hunk 1 above)
Comment (dbetka) [reply client/webpack/plugins/app-console-frame.js:1]: Yes, I'll remove it

File: client/webpack/plugins/app-console-frame.js (hunk 2 above)
Comment (dbetka) [reply client/webpack/plugins/app-console-frame.js:4]: Yes, I'll remove it

**Review by Bedpaw — APPROVED** [review]

**Comment by Bedpaw** [comment]
Project built fine after changes, good job with removing dependencies to wdk :+1:

**Pull request merged**